docs/en/api/POSIX-API.md

```diff
@@ -140,6 +163,37 @@ pid mount_point alluxio_path
 
 ## Advanced Setup
 
+### Fuse on worker process
+
+Unlike standalone Fuse which you can mount at any time without Alluxio worker involves,
+the embedded Fuse has the exact same life cycle as the worker process it embeds into.
+When the worker starts, the Fuse is mounted based on worker configuration.
+When the worker ends, the embedded Fuse is unmounted automatically.
+If you want to modify your Fuse mount, change the configuration and restart the worker process.
+
+Enable FUSE on worker by setting `alluxio.worker.fuse.enabled` to `true` in the `${ALLUXIO_HOME}/conf/alluxio-site.properties` for workers:
+
+```
+alluxio.worker.fuse.enabled=true
+```
+
+All the FUSE configuration needs to be provided as the worker configuration.
+
+```
+alluxio.worker.fuse.mount.alluxio.path=<alluxio_path>
+alluxio.worker.fuse.mount.point=<mount_point>
```

Comment:
this is required,  right? then put it togetther with `alluxio.worker.fuse.enabled` above

Reply:
Yeah, it's required, move it upward